authservice/config/config.go

```diff
@@ -0,0 +1,59 @@
+package config
+
+import (
+	"fmt"
+	"github.com/caarlos0/env/v11"
+	"log"
+)
+
+type Config struct {
+	// Database
+	DBHost     string `env:"DB_HOST,required"`
+	DBPort     int    `env:"DB_PORT" envDefault:"5432"`
+	DBUser     string `env:"DB_USER,required"`
+	DBPassword string `env:"DB_PASSWORD,required"`
+	DBName     string `env:"DB_NAME,required"`
+
+	// JWT
+	JWTAccessSecret  string `env:"JWT_ACCESS_SECRET,required"`
+	JWTRefreshSecret string `env:"JWT_REFRESH_SECRET,required"`
+
+	// Service
+	GRPCPort    int    `env:"GRPC_PORT" envDefault:"50051"`
+	Environment string `env:"ENVIRONMENT" envDefault:"development"`
+}
+
+func Load() *Config {
```

Comment:
```suggestion
func Load() *Config, error {
```
log.Fatalf в одном месте где вызов Load

Reply:
<img width="878" height="530" alt="image" src="https://github.com/user-attachments/assets/31549835-2954-4dc2-9ecc-65fb4d10d4ab" />
я же написал пример